pkg/crosscluster/physical/alter_replication_job_test.go

```diff
@@ -597,3 +597,22 @@ func TestAlterTenantStartReplicationAfterRestore(t *testing.T) {
 	srcTime := srv.Clock().Now()
 	replicationtestutils.WaitUntilReplicatedTime(t, srcTime, db, catpb.JobID(ingestionJobID))
 }
+
+func TestAlterReplicationJobErrors(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	ctx := context.Background()
+
+	srv, sqlDB, _ := serverutils.StartServer(t, base.TestServerArgs{
```

Comment:
are there really no other tests in this file that we could add a case to rather than starting a new server?